client/dialer.go

```diff
@@ -55,31 +53,18 @@ func (c *HostClient) makeDialer(superProxy *superproxy.SuperProxy,
 	//set https tls config
 	switch reqType {
 	case requestDirectHTTP:
-		return dialerWrapper(transport.Dial(targetWithPort))
+		fallthrough
 	case requestDirectHTTPS:
```

Comment:
this is a traditional http client, its the dialer's responsibility to make the TLS connection for client, you should NOT remove the dialTLS here

Reply:
as for tunnel use, you should declare a new `AcquireRawConn` for it with this type of code, you can alternatively  add a `isRaw` switch for this func.